### ISSUE TITLE
Check_MK integration page gives 500 error in 2.5.0

### ISSUE THREAD

**Issue by penzoiders** [zammad/zammad#2100]
### Infos:

* Used Zammad version: zammad-2.5.0-1529668954.708c86ba.centos7.x86_64

* Installation method (source, package, ..): package repository
* Operating system: CentOS Linux release 7.5.1804 (Core) 
* Database + version: postgresql-server-9.2.23-3.el7_4.x86_64
* Elasticsearch version: elasticsearch-5.6.10-1.noarch
* Browser + version: Google Chrome 67.0.3396.87

### Expected behavior:

* not getting a 500 when visiting the Check_MK integration page (usually I see the latest calls to debug them)
* possibly getting it to work even if accented characters appear in the API call

### Actual behavior:

* getting a 500 errror
![selezione_065](https://user-images.githubusercontent.com/7568057/41969110-b6a0ea76-7a06-11e8-97de-a6271b9d5845.png)

```
I, [2018-06-27T07:58:04.020106 #1307]  INFO -- : Started GET "/api/v1/http_logs/check_mk?limit=50&_=1529941048387" for 185.121.33.41 at 2018-06-27 07:58:04 +0000
I, [2018-06-27T07:58:04.025242 #1307]  INFO -- : Processing by HttpLogsController#index as JSON
I, [2018-06-27T07:58:04.025331 #1307]  INFO -- :   Parameters: {"limit"=>"50", "_"=>"1529941048387", "facility"=>"check_mk"}
E, [2018-06-27T07:58:04.089893 #1307] ERROR -- : "\xC3" from ASCII-8BIT to UTF-8 (Encoding::UndefinedConversionError)
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/object/json.rb:36:in `encode'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/object/json.rb:36:in `to_json'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/object/json.rb:36:in `to_json'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/json/encoding.rb:55:in `to_json'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/json-2.1.0/lib/json/common.rb:224:in `generate'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/json-2.1.0/lib/json/common.rb:224:in `generate'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/json/encoding.rb:100:in `stringify'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/json/encoding.rb:33:in `encode'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/json/encoding.rb:20:in `encode'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/object/json.rb:39:in `to_json'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/renderers.rb:156:in `block in <module:Renderers>'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/renderers.rb:149:in `block in _render_to_body_with_renderer'
/opt/zammad/vendor/ruby-2.4.4/lib/ruby/2.4.0/set.rb:324:in `each_key'
/opt/zammad/vendor/ruby-2.4.4/lib/ruby/2.4.0/set.rb:324:in `each'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/renderers.rb:145:in `_render_to_body_with_renderer'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/renderers.rb:141:in `render_to_body'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/abstract_controller/rendering.rb:24:in `render'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/rendering.rb:36:in `render'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
/opt/zammad/vendor/ruby-2.4.4/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/core_ext/benchmark.rb:12:in `ms'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:44:in `block in render'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activerecord-5.1.5/lib/active_record/railties/controller_runtime.rb:29:in `cleanup_view_runtime'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:43:in `render'
/opt/zammad/app/controllers/application_controller/renders_models.rb:152:in `model_index_render_result'
/opt/zammad/app/controllers/http_logs_controller.rb:14:in `index'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/abstract_controller/base.rb:186:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/rendering.rb:30:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/abstract_controller/callbacks.rb:20:in `block in process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/callbacks.rb:131:in `run_callbacks'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/abstract_controller/callbacks.rb:19:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/rescue.rb:20:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/notifications.rb:166:in `block in instrument'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/notifications.rb:166:in `instrument'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/instrumentation.rb:30:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal/params_wrapper.rb:252:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activerecord-5.1.5/lib/active_record/railties/controller_runtime.rb:22:in `process_action'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/abstract_controller/base.rb:124:in `process'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.5/lib/action_view/rendering.rb:30:in `process'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal.rb:189:in `dispatch'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_controller/metal.rb:253:in `dispatch'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/routing/route_set.rb:49:in `dispatch'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/routing/route_set.rb:31:in `serve'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/journey/router.rb:50:in `block in serve'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/journey/router.rb:33:in `each'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/journey/router.rb:33:in `serve'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/routing/route_set.rb:844:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:190:in `call!'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/strategy.rb:168:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/omniauth-1.7.1/lib/omniauth/builder.rb:63:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/etag.rb:25:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/conditional_get.rb:25:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/head.rb:12:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:232:in `context'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:226:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/cookies.rb:613:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/callbacks.rb:26:in `block in call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/callbacks.rb:97:in `run_callbacks'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/callbacks.rb:24:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/railties-5.1.5/lib/rails/rack/logger.rb:36:in `call_app'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/railties-5.1.5/lib/rails/rack/logger.rb:24:in `block in call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/tagged_logging.rb:69:in `block in tagged'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/tagged_logging.rb:26:in `tagged'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/tagged_logging.rb:69:in `tagged'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/railties-5.1.5/lib/rails/rack/logger.rb:24:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/request_id.rb:25:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/method_override.rb:22:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/runtime.rb:22:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.5/lib/active_support/cache/strategy/local_cache_middleware.rb:27:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/executor.rb:12:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.5/lib/action_dispatch/middleware/static.rb:125:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/sendfile.rb:111:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/railties-5.1.5/lib/rails/engine.rb:522:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/puma-3.11.0/lib/puma/configuration.rb:225:in `call'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/puma-3.11.0/lib/puma/server.rb:624:in `handle_request'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/puma-3.11.0/lib/puma/server.rb:438:in `process_client'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/puma-3.11.0/lib/puma/server.rb:302:in `block in run'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/puma-3.11.0/lib/puma/thread_pool.rb:120:in `block in spawn_thread'
/opt/zammad/vendor/bundle/ruby/2.4.0/gems/logging-2.2.2/lib/logging/diagnostic_context.rb:474:in `block in create_with_logging_context'
I, [2018-06-27T07:58:04.090690 #1307]  INFO -- : Completed 500 Internal Server Error in 65ms (Views: 0.2ms | ActiveRecord: 3.5ms)
```

### Steps to reproduce the behavior:

*Enable Check_MK integration, configure check_mk to send alerts via custom script, update from 2.4.0 to 2.5.0
* I don’t know if this was introduced by the update or if just broke at a certain point, used to work when set up… Now I wanted to investigate on how to handle HOST_STATUS along the already working SERVICE_STATUS notifications… I went to the integration page to check the contents of the API calls but got the error.

Yes I'm sure this is a bug and no feature request or a general question.

This bug comes from this tech support community thread: https://community.zammad.org/t/check-mk-integration-page-gives-500-error-in-2-5-0/897

**Comment by penzoiders** [comment]
Probably there’s an accented à in the body of the Check_MK service notification due to the Italian language of the Windows monitored host.

`remoto \xC3\xA8 stato emesso da unAutorit\xC3\xA0`
comes from this string in check_mk
`remoto è stato emesso da un'Autorità`

heres the anonymized output of 
```ruby
HttpLog.where(facility: 'check_mk').order('created_at DESC').limit(50).each do |entry|
  begin
    entry.to_json
  rescue => e
    p entry
  end
end
```
OUTPUT:

```
#<HttpLog id: 225, direction: "in", facility: "check_mk", method: "POST", url: "https://zammad.fqdn.com/api/v1/integration/...", status: "200", ip: "xIP.xIP.xIP.xIP", request: {"content"=>"HTTP_VERSION: HTTP/1.0\nHTTP_HOST: zammad.fqdn.com\nHTTP_CLIENT_IP: xIP.xIP.xIP.xIP\nHTTP_X_FORWARDED_FOR: xIP.xIP.xIP.xIP\nHTTP_X_FORWARDED_PROTO: https\nHTTP_CONNECTION: close\nHTTP_USER_AGENT: curl/7.19.7 (x86_64-redhat-linux-gnu) libcurl/7.19.7 NSS/3.27.1 zlib/1.2.3 libidn/1.18 libssh2/1.4.2\nHTTP_ACCEPT: */*\n\n------------------------------9b34f2a406fa\r\nContent-Disposition: form-data; name=\"event_id\"\r\n\r\n16599\r\n------------------------------9b34f2a406fa\r\nContent-Disposition: form-data; name=\"host\"\r\n\r\nsql01.testing.fqdn.com\r\n------------------------------9b34f2a406fa\r\nContent-Disposition: form-data; name=\"service\"\r\n\r\nLog System\r\n------------------------------9b34f2a406fa\r\nContent-Disposition: form-data; name=\"state\"\r\n\r\nCRITICAL\r\n------------------------------9b34f2a406fa\r\nContent-Disposition: form-data; name=\"text\"\r\n\r\nCRIT - 8 CRIT, 1 WARN messages (Last worst: Jun 18 11:28:53 0.36882 Schannel Il certificato ricevuto dal server remoto \xC3\xA8 stato emesso da unAutorit\xC3\xA0 di certificazione non disponibile nellelenco locale. Non \xC3\xA8 pertanto possibile convalidare i dati contenuti nel certificato. La richiesta di connessione SSL non \xC3\xA8 riuscita. Il certificato server \xC3\xA8 incluso nei dati allegati.)\r\n------------------------------9b34f2a406fa--\r\n", "content_type"=>"multipart/form-data; boundary=----------------------------9b34f2a406fa", "content_encoding"=>nil, "source"=>"curl/7.19.7 (x86_64-redhat-linux-gnu) libcurl/7.19.7 NSS/3.27.1 zlib/1.2.3 libidn/1.18 libssh2/1.4.2"}, response: {"code"=>"200", "content"=>"X-Frame-Options: SAMEORIGIN\nX-XSS-Protection: 1; mode=block\nX-Content-Type-Options: nosniff\nContent-Type: application/json; charset=utf-8\n\n{\"result\":\"ticket already open, added note\",\"ticket_ids\":[46]}", "content_type"=>nil, "content_encoding"=>nil, "source"=>nil}, updated_by_id: 1, created_by_id: 1, created_at: "2018-06-18 14:10:57", updated_at: "2018-06-18 14:10:57">
```

**Comment by thorsteneckel** [comment]
Hi @rlue - We need your charset conversion power over here 💪 

Looks like we need to encode the input before storing it (and add a migration to fix existing data 😱 ) over here:

https://github.com/zammad/zammad/blob/a9239997effa2a07c6c41270aecfbc305be63576/app/controllers/application_controller/logs_http_access.rb#L19-L37

But please verify it by writing a RSpec controller test first. Thanks!

**Comment by rlue** [comment]
@thorsteneckel, I'm thinking it would be simpler (and faster to test) if we simply added a `before_save` callback on the `HttpLog` model. Sound okay to you?

@penzoiders, could I trouble you to share one last detail? What's the output of the following command?

```
$ rails r 'p HttpLog.find(225).request["content"].encoding'
```

---

**EDIT:** Scratch that; ActiveRecord doesn't save the encoding of individual strings in `HttpLog` attributes (and on second thought, string encoding should be determined by database configuration values).

In fact, the string stored in the database is valid UTF-8; you can enter it (double-quoted) in a Ruby console and see the correct output:

```ruby
>> "Schannel Il certificato ricevuto dal server remoto \xC3\xA8 stato emesso da unAutorit\xC3\xA0 di certificazione non disponibile nellelenco locale. Non \xC3\xA8 pertanto possibile convalidare i dati contenuti nel certificato. La richiesta di connessione SSL non \xC3\xA8 riuscita. Il certificato server \xC3\xA8 incluso nei dati allegati."
=> "Schannel Il certificato ricevuto dal server remoto è stato emesso da unAutorità di certificazione non disponibile nellelenco locale. Non è pertanto possibile convalidare i dati contenuti nel certificato. La richiesta di connessione SSL non è riuscita. Il certificato server è incluso nei dati allegati."
```

For some reason, it looks like something in the `render json: xxx` call treats the string as binary (ASCII-8BIT) and tries to convert it back to Unicode. I'll have to take a closer look. More details to come.

**Comment by penzoiders** [comment]
> @penzoiders, could I trouble you to share one last detail? What's the output of the following command?
> `$ rails r 'p HttpLog.find(225).request["content"].encoding'`

here it is:

```
[root@zammad ~]# zammad run rails r 'p HttpLog.find(225).request["content"].encoding'

#<Encoding:ASCII-8BIT>
```

**Comment by rlue** [comment]
Oh, that's not what I was expecting. It's possible this is a database configuration issue then — what value do you have set for `encoding` in `config/database.yml`? (It should be `utf8`; [here's a sample config file for reference](https://github.com/zammad/zammad/blob/develop/config/database/database.yml#L9).)

---

**EDIT:** Sorry, upon closer inspection, this is probably still an issue with Zammad. Working on a fix now.

**Comment by thorsteneckel** [comment]
> @thorsteneckel, I'm thinking it would be simpler (and faster to test) if we simply added a before_save callback on the HttpLog model. Sound okay to you?

@rlue  You're absolutely right!

**Comment by penzoiders** [comment]
sorry for late reply:
```
[root@zammad ~]# cat /opt/zammad/config/database.yml
production:
  adapter: postgresql
  database: zammad
  pool: 50
  timeout: 5000
  encoding: utf8
  username: zammad
  password: 
```

**Comment by rlue** [comment]
@penzoiders it's all fixed now, and should be pushed as a 2.6 release on hub.docker.com soon. (A 2.5 release will be available in the next few days.)
